packages/app/src/debug/DebugFailedTest.vue

```diff
@@ -7,7 +7,7 @@
       size="16"
       status="failed"
       data-cy="failed-icon"
-      class="isolate"
+      class="min-w-16px isolate"
```

Comment:
I did not test this, but I think another option that would work is to use `flex-shrink-0` to tell the flexbox not the reduce the size of this child element.

Reply:
I think that would work just fine too.

Reply:
I am with @warrensplayer on this one. If we want to avoid shrinking, it's better to use `flex-shrink-0` if it works.
I am afraid of the side effects od doing this in the design system. specially if we want to use the `size="16"` model with a width of 8px for example.